src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java

```diff
@@ -35,7 +35,7 @@
 import uk.gov.hmcts.reform.civil.model.sdo.DisposalHearingDisclosureOfDocuments;
 import uk.gov.hmcts.reform.civil.model.sdo.DisposalHearingFinalDisposalHearing;
 import uk.gov.hmcts.reform.civil.model.sdo.DisposalHearingJudgesRecital;
-import uk.gov.hmcts.reform.civil.model.sdo.DisposalHearingMedicalEvidence;
+//import uk.gov.hmcts.reform.civil.model.sdo.DisposalHearingMedicalEvidence;
```

Comment:
if it's not needed, it's better to remove it rather than commenting it out. It's in git if we need it again